### ISSUE TITLE
fix: menu component not adapting colors properly

### ISSUE THREAD

**Issue by radium-v** [microsoft/fast#5060]
# 🐛 Bug Report

The Menu, Menu Item, and Listbox (Listbox Element, Select, and Combobox) don't properly adapt their colors when `fill-color` changes.

## 💻 Repro or Code Sample

In `fast-components`:
1. Modify the file `.storybook/preview.js` to set the value for the `fillColor`:
    ```js
    import * as FAST from "../src/index-rollup";
    import { parseColor } from "@microsoft/fast-colors";
    FAST.fillColor.setValueFor(document.body, FAST.SwatchRGB.from(parseColor("#FFFFFF")));
    // ...
    ```
2. Run `yarn start` to start Storybook.
3. Navigate to the Menu story in the sidebar.

## 🤔 Expected Behavior

The Menu should be rendered with a light color scheme.

## 😯 Current Behavior

Menu:
![Screenshot 2021-08-13 150341](https://user-images.githubusercontent.com/863023/129423071-3a6138b3-3373-4ef1-bd0b-77c768bdf8e8.png)

Listbox:
![Screenshot 2021-08-13 151435](https://user-images.githubusercontent.com/863023/129423171-03b9fa23-cc2c-49e5-b0a9-5fa19804e47d.png)

Select:
![image](https://user-images.githubusercontent.com/863023/129423294-b2b81e63-39d0-405e-8332-edce817bace2.png)

Combobox:
![image](https://user-images.githubusercontent.com/863023/129423336-dd507b5e-c88e-4ed1-99ad-6f45efb06e5c.png)

## 🔦 Context

These components all use `neutralLayerFloating`, as well as other `neutral` values that don't update when in light mode.

## 🌍 Your Environment

* OS & Device: Windows 10 PC
* Browser Microsoft Edge
* Version 92.0.902.73

**Comment by chrisdholt** [comment]
@radium-v @bheston is this still an issue we are tracking?

**Comment by radium-v** [comment]
Fixed in #5091
